project/AkkaBuild.scala

```diff
@@ -400,7 +137,20 @@ object AkkaBuild extends Build {
     testOptions in Test += Tests.Argument("-oDF"),
 
     // don't save test output to a file
-    testListeners in (Test, test) := Seq(TestLogger(streams.value.log, {_ => streams.value.log }, logBuffered.value)),
```

Comment:
@johanandren It seems that this has broken proper logging of scalatest logging. Can you explain why that change was necessary and what it does? ;)